ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx

```diff
@@ -191,11 +191,12 @@ export default function Page(props) {
               onChange={(e) => setKey(e.target.value)}
               disabled={disabled}
             />
-            {!exactProperty && key !== column ? (
+            {!exactProperty &&
+            columnSpeculation[column].suggestedPropertyKey !== column ? (
```

Comment:
We could probably pre-store `columnSpeculation[column].suggestedPropertyKey` as a const var at top of the function so it's not too wordy in the component rendering

Reply:
Thanks! Definitely looks cleaner